### PR TITLE
docs(components): [page-header] use new display tag

### PR DESCRIPTION
**Pull request by wzc520pyfm**
Related: #10754 

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: docs/en-US/component/page-header.md [hunk 1]
```diff
@@ -98,27 +98,29 @@ The component is consisted of these parts
...
-| breadcrumb | breadcrumb content |
-| default    | main content       |
+## API
+
+### Attributes
+
+| Name    | Description                   | Type                     | Default |
+| ------- | ----------------------------- | ------------------------ | ------- |
+| icon    | icon component of page header | ^[string] / ^[Component] | Back    |
+| title   | main title of page header     | ^[string]                | Back    |
```
Comment (chenxch) [inline docs/en-US/component/page-header.md:108]: title doesn't seem to have a default value.

Comment (wzc520pyfm) [reply docs/en-US/component/page-header.md:108]: Although the default value is not specified in prop, the default value is specified in the template.
Do I need to change `Back` to `''`? 
image

Comment (chenxch) [reply docs/en-US/component/page-header.md:108]: Yes, the default value is subject to the declaration in props, and the default value of the template can be considered in the description.

Comment (wzc520pyfm) [reply docs/en-US/component/page-header.md]: updated

**Review by ryuhangyeong — APPROVED** [review]

**Pull request merged**